tests/networkpolicy_test.go

```diff
@@ -147,14 +141,18 @@ var _ = Describe("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 		It("[test_id:1513] should be successful to reach vmia from vmib", func() {
 			By("Connect vmia from vmib in same namespace")
-			ip := vmia.Status.Interfaces[0].IP
-			assertPingSucceed(ip, vmib)
+			Expect(vmia.Status.Interfaces[0].IPs).NotTo(BeEmpty())
+			for _, ip := range vmia.Status.Interfaces[0].IPs {
+				assertPingSucceed(ip, vmib)
+			}
```

Comment:
Please encapsulate this into a function, then use it.

Reply:
done, if needed will squash the commit to other one